src/index.js

```diff
@@ -9,6 +9,7 @@ var EventEmitter = require('events').EventEmitter;
  *
  * @param {Object} options
  *   @property {String} redis Redis connection string.
+ *   @property {Object} redisConnection Pre-existing Redis connection.
```

Comment:
If `redis` is an object, it could be a configuration object for the redis connection, as opposed to a connection itself - is there a good way to detect which?